website/templates/donations/view-donation.html

```diff
@@ -48,6 +48,17 @@
           <span class="mdc-list-item__meta">{{ donation.time_created }}</span>
         </li>
       </ul>
+    </div>
+    <!-- Buttons -->
+    <div class="emblem-campaign-details mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
+      <!-- View Logs button -->
+      <a href="https://console.cloud.google.com/logs/query;query=resource.type%3D%22cloud_run_revision%22;cursorTimestamp={{ donation.time_created.strftime('%Y-%m-%dT%H:%M:%S.%fZ') }}" class="mdc-button mdc-button--raised" target=”_blank”>
```

Comment:
Time is good, but what I'm hoping for is pulling the trace context from the HTTP request and linking to logs pre-filtered to that as a sort of request ID.